### PR TITLE
B2B TOTPs + General SDK Bump

### PR DESCRIPTION
**Pull request by ollie-stytch**
- Added B2B TOTP endpoints 
- General SDK Bump

### CODE REVIEW

**Review by taronish-stytch — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "10.0.1",
+  "version": "10.0.2",
```
Comment (logan-stytch) [inline package.json:3]: This definitely should've been a MINOR update, btw. It added an entirely new product. I'd actually recommend we put up another PR as 10.1.0 with no other changes just so people know 10.1 added new features
